web/src/components/organisms/Asset/convertAsset.ts

```diff
@@ -21,6 +22,7 @@ export const convertAsset = (GQLAsset: GQLAsset | undefined): Asset | undefined
     url: GQLAsset.url,
     threadId: GQLAsset.thread?.id ?? "",
     comments: GQLAsset.thread?.comments?.map(comment => convertComment(comment)) ?? [],
+    archiveExtractionStatus: GQLAsset.archiveExtractionStatus as ArchiveExtractionStatus,
```

Comment:
Is this necessary,
`GQLAsset.archiveExtractionStatus as ArchiveExtractionStatus`
 as it could take `undefined` as a value?

Reply:
When I remove this I get the following error message:
Type 'Maybe<ArchiveExtractionStatus> | undefined' is not assignable to type 'ArchiveExtractionStatus'.
  Type 'null' is not assignable to type 'ArchiveExtractionStatus'.